cooltools/sandbox/cooler_filters/pixel_filter_util.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+from toolz import curry
+import cooltools
+import cooler
+import functools
+from multiprocessing import Pool
+import logging
+from cooltools.lib.common import pool_decorator
+
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s : %(message)s")
+logger = logging.getLogger("data_util")
+logger.propagate = False  # Disable propagation to the root logger
+
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+ch.setFormatter(formatter)
+logger.addHandler(ch)
+
+@curry
+def cis_total_ratio_filter(clr, thres=0.5):
```

Comment:
I would go with just `threshold=0.5` instead of thres, because it may be easier to remember (e.g. I would forget if I should have thres or thresh)